src/Web/Sfa.Das.Sas.Web/Controllers/ProviderController.cs

```diff
@@ -177,22 +178,31 @@ public sealed class ProviderController : Controller
         }
 
         [HttpGet]
-        public ActionResult Detail(ProviderDetailQuery criteria)
+        public async Task<ActionResult> ProviderDetail(long id)
+        {
+            var response = await _mediator.SendAsync(new ProviderDetailQuery { ukPrn = id});
+            var viewModel = ProviderDetailViewModelMapper.GetProviderDetailViewModel(response.Provider);
+
+            return View(viewModel);
+        }
+
+        [HttpGet]
+        public ActionResult Detail(ApprenticeshipProviderDetailQuery criteria)
         {
             var response = _mediator.Send(criteria);
 
             switch (response.StatusCode)
             {
-                case DetailProviderResponse.ResponseCodes.ApprenticeshipProviderNotFound:
-                    _logger.Warn($"404 - Cannot find provider: ({criteria.Ukprn}) for apprenticeship product: ({criteria.FrameworkId ?? criteria.StandardCode}) with location: ({criteria.LocationId})");
+                case ApprenticeshipProviderDetailResponse.ResponseCodes.ApprenticeshipProviderNotFound:
+                    _logger.Warn($"404 - Cannot find provider: ({criteria.UkPrn}) for apprenticeship product: ({criteria.FrameworkId ?? criteria.StandardCode}) with location: ({criteria.LocationId})");
                     return new HttpStatusCodeResult(HttpStatusCode.NotFound);
 
-                case DetailProviderResponse.ResponseCodes.InvalidInput:
-                    _logger.Warn($"400 - Bad Request: {criteria.Ukprn}");
+                case ApprenticeshipProviderDetailResponse.ResponseCodes.InvalidInput:
+                    _logger.Warn($"400 - Bad Request: {criteria.UkPrn}");
                     return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
             }
 
-            var viewModel = _mappingService.Map<DetailProviderResponse, ApprenticeshipDetailsViewModel>(response, opt => opt
+            var viewModel = _mappingService.Map<ApprenticeshipProviderDetailResponse, ApprenticeshipDetailsViewModel>(response, opt => opt
                                 .AfterMap((src, dest) =>
                                 {
                                     dest.SurveyUrl = _settings.SurveyUrl.ToString();
```

Comment:
For the other provider page they use the Shared model for this

```html
<a href="@Model.SurveyUrl" target="_blank" class="button">Take the survey</a>
```

I don't have a strong opinion on this

Reply:
This was just a renaming exercise to distinguish ProviderDetails from Apprenticeship pathways from ProviderDetails in the new sitemap code, and not code I wrote/changed, other than renaming.  I'll leave unless strong requirement.